src/patternfly/components/Divider/divider.scss

```
@@ -1,98 +1,83 @@
 // @debug $divider; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
-$pf-v5-c-divider--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
+$pf-v5-c-divider--breakpoint-map: build-breakpoint-map("sm", "md", "lg", "xl", "2xl");
 $pf-v5-c-divider--spacer-map: build-spacer-map("none", "xs", "sm", "md", "lg", "xl", "2xl", "3xl");
 
+// * Divider horizontal
 @mixin pf-v5-c-divider--m-horizontal {
-  --#{$divider}--Display: var(--#{$divider}--m-horizontal--Display);
-  --#{$divider}--FlexDirection: var(--#{$divider}--m-horizontal--FlexDirection);
-  --#{$divider}--after--Width: var(--#{$divider}--m-horizontal--after--Width);
-  --#{$divider}--after--Height: var(--#{$divider}--m-horizontal--after--Height);
-
-  width: 100%; // update to var breaking change
-  height: auto; // update to var breaking change
-
-  &::after {
-    flex-basis:
-      calc(
-        var(--#{$divider}--after--FlexBasis) - calc(var(--#{$divider}--after--Inset) * 2)
-      ); // update at breaking change
-  }
+  flex-direction: row;
+  width: 100%;
+  height: var(--#{$divider}--Size);  // apply size to width in horizontal orientation
```

nit, imo the comments probably aren't necessary

```suggestion
  height: var(--#{$divider}--Size);  // apply size to height in horizontal orientation
```